### PR TITLE
Fixed Issue #1

### PR DESCRIPTION
**Pull request by xDoges**
Fixing the dog issue aka #2

### CODE REVIEW

File: src/commands/misc/dog.js [hunk 1]
```diff
@@ -11,9 +11,18 @@ module.exports = class Dog extends Command {
...
-      const embed = this.client.getDefaultEmbed(message.author)
-      embed.setImage(r.body.url).setDescription('Here is your dog <:DoggoF:445701839564963840>')
-      message.channel.send({embed})
+      var notsupported = ['.mp4']
+      if (!r.body.url.endsWith(notsupported)) {
+        const embed = this.client.getDefaultEmbed(message.author)
+        embed.setImage(r.body.url).setDescription('Here is your dog <:DoggoF:445701839564963840>')
+        message.channel.send({embed})
+      } else {
+        request.get('https://random.dog/woof.json').then(r => {
```
Comment (DoctorVWA) [inline src/commands/misc/dog.js]: I belive a recursive function checking if the file aren't a mp4 is better here, it totally can gets another .mp4 after the first one

Comment (xDoges) [reply src/commands/misc/dog.js]: i can fix this with a thing

File: src/commands/misc/dog.js [hunk 2]
```diff
@@ -18,9 +18,13 @@ module.exports = class Dog extends Command {
         message.channel.send({embed})
       } else {
         request.get('https://random.dog/woof.json').then(r => {
-          const embed = this.client.getDefaultEmbed(message.author)
-          embed.setImage(r.body.url).setDescription('Here is your dog <:DoggoF:445701839564963840>')
-          message.channel.send({embed})
+          if (r.body.url.endsWith(notsupported)) {
+            request.get('https://random.dog/woof.json').then(r => {
```
Comment (DoctorVWA) [inline src/commands/misc/dog.js]: The problem wasn't the if, in the second request it just can get another .mp4 and will return another blank embed

Comment (xDoges) [reply src/commands/misc/dog.js]: lmao, 3 times once, i tried like, 100 times to request and, thats impossible

Comment (davipatury) [reply src/commands/misc/dog.js]: The use of recursive function in this case prevents repeated code and the possibillity of requesting three MP4 in a row.

File: src/commands/misc/dog.js [hunk 3]
```diff
@@ -11,9 +11,12 @@ module.exports = class Dog extends Command {
...
-      embed.setImage(r.body.url).setDescription('Here is your dog <:DoggoF:445701839564963840>')
-      message.channel.send({embed})
+      var notsupported = ['.mp4']
+      if (!r.body.url.endsWith(notsupported)) {
+        const embed = this.client.getDefaultEmbed(message.author)
+        embed.setImage(r.body.url).setDescription('Here is your dog <:DoggoF:445701839564963840>')
+        message.channel.send({embed})
+      } else this.run(message)
     })
     message.channel.stopTyping()
```
Comment (pedrofracassi) [inline src/commands/misc/dog.js]: This should be inside of the request.then method.

Comment (davipatury) [reply src/commands/misc/dog.js]: You should probably use async function too, it makes the code cleaner.

**Pull request merged**